app/javascript/components/util/InfoTooltip.js

```diff
@@ -21,6 +21,13 @@ const TOOLTIP_TEXT = {
     </div>
   ),
 
+  editingWorkflow: (
```

Comment:
is there a reason we can't just call this `workflow` to match the rest of enrollment fields?

Reply:
or `disabledWorkflow`